content/1.2/docs/serverless-workflows/troubleshooting.md

```diff
@@ -93,36 +93,8 @@ a CA which is not available to the workflow. The error in the workflow pod log u
 ### Problem: Workflow installed in a different namespace than Sonataflow services fails to start
 
 **Solution:**
-When deploying a workflow in a namespace other than the one where Sonataflow services are running (e.g., sonataflow-infra), there are two essential steps to follow if persistence is required for the workflow:
-1. Create a Secret with PostgreSQL Credentials in the target namespace:
-    1. The workflow needs to create its own schema in PostgreSQL. To enable this, you must create a secret containing the PostgreSQL credentials in the same namespace as the workflow.
-2. Configure the Namespace Attribute:
-    1. Add the namespace attribute under the serviceRef where the PostgreSQL server is deployed.
-
-**Example Configuration:**
-```
-apiVersion: sonataflow.org/v1alpha08
-kind: SonataFlow
-...
-spec:
-  ...
-  persistence:
-    postgresql:
-      secretRef:
-        name: sonataflow-psql-postgresql
-        passwordKey: postgres-password
-        userKey: postgres-username
-      serviceRef:
-        databaseName: sonataflow
-        databaseSchema: greeting
-        name: sonataflow-psql-postgresql
-        namespace: <postgresql-namespace>
-        port: 5432
-```
-In this configuration:
-* Replace <postgresql-namespace> with the namespace where the PostgreSQL server is deployed.
-
-By following these steps, the workflow will have the necessary credentials to access PostgreSQL and will correctly reference the service in a different namespace.
+When deploying a workflow in a namespace other than the one where Sonataflow services are running (e.g., sonataflow-infra), there are essential steps to follow to enable persistence and connectivity for the workflow. See the following [steps](https://github.com/parodos-dev/orchestrator-helm-operator/blob/gh-pages/README.md#additional-workflow-namespaces).
+
```

Comment:
Regarding the "Performance Issues" section (below) since there is only a TBD, should we remove it?

Reply:
yes, I'm removing those TBD in the next PR (part of a different Jira issue :-) )